charts/fission-core/templates/nonhelm-install-job.yaml

```diff
@@ -26,6 +26,16 @@ spec:
       restartPolicy: Never
       containers:
         - name: post-install-job
-          image: "fission/alpinecurl"
-          command: ["sh", "-c", "/usr/bin/curl -m 5 -H \"Content-Type: application/json\" -X POST -d '{\"type\": \"yaml-post-install\", \"chartName\": \"{{ .Chart.Name }}\", \"chartVersion\": \"{{ .Chart.Version }}\"}' https://g.fission.io/analytics || true"]
+        {{- if .Values.imageTag }}
+          image: {{ .Values.postInstallReportImage }}:{{ .Values.imageTag }}
+        {{- else }}
+          image: {{ .Values.postInstallReportImage }}
+        {{- end }}
+          imagePullPolicy: {{ .Values.pullPolicy }}
+          command: [ "/reporter" ]
+          args: ["event", "-c", "fission-use", "-a", "helm-post-install", "-l", "{{ .Chart.Name }}-{{ .Chart.Version }}"]
```

Comment:
If this file is used for non helm install - then should the `helm-post-install` not be `yaml-post-install` or something similar?